release/nightly_tests/dataset/image_loader_microbenchmark.py

```diff
@@ -10,6 +10,10 @@
 import pandas as pd
 import json
 
+# HF Dataset.
+from datasets import load_dataset
```

Comment:
maybe move this import inline the function? same for mosaicml streaming.
sometimes I needed to manually run the script on an environment with no mosaicml, and I had to modified the script.